conf/radiusd/auth.conf.example

```diff
@@ -10,11 +10,44 @@ listen {
     virtual_server = packetfence
 }
 
-%%listen%%
+[% FOREACH ip IN listen_ips %]
+listen {
+        ipaddr = [% ip %]
+        port = 0
+        type = auth
+        virtual_server = packetfence
+}
+
+listen {
+        ipaddr = [% ip %]
+        port = 2083
+        type = auth+acct
+        proto = tcp
+        virtual_server = packetfence
+
+        limit {
+              max_connections = 16
+              lifetime = 0
+              idle_timeout = 30
+        }
+
+        tls {
```

Comment:
Is there any way to have a single configuration for tls instead of here and in the eap module config?